.github/workflows/crowdin-contributors.yml

```
@@ -0,0 +1,41 @@
+name: Crowdin Contributors Action
+
+on:
+  schedule:
+    - cron: '0 0 * * 1' # https://crontab.guru/#0_0_*_*_1
```

```suggestion
    - cron: '0 0 * * 1' # 'https://crontab.guru/#0_0_*_*_1'
```

I would put quotes round this URL as well as it contains a '*' which has special meaning in YAML and, I suspect, might cause problems. I know it's commented, but you can never be too careful with YAML.